common/src/IO/MapReader.cpp

```diff
@@ -159,6 +160,61 @@ namespace TrenchBroom {
                 ));
         }
 
+        // helper methods
+
+        void MapReader::createNodes(ParserStatus& status) {
+            loadBrushes(status);
```

Comment:
Could this function not return the loaded brushes instead of storing them in a member variable? They seam to be used in this function an in `createNode`, to which they could be passed.